SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMovement/SpatialTestCharacterMovement.cpp

```diff
@@ -145,7 +145,7 @@ void ASpatialTestCharacterMovement::BeginPlay()
 
 
 	// Universal clean-up step to delete the TriggerBox from all connected clients and servers		
-	AddUniversalStep(TEXT("SpatialTestCharacterMovementUniversalCleanUp"), nullptr, [](ASpatialFunctionalTest* NetTest) {
+	AddStep(TEXT("SpatialTestCharacterMovementUniversalCleanUp"), FWorkerDefinition::AllServers, nullptr, [](ASpatialFunctionalTest* NetTest) {
```

Comment:
I think this should be `FWorkerDefinition::AllWorkers` as is the first step in the test, since it requires both clients and servers to delete the spawned TriggerBox.

Reply:
Nice catch, thanks, was auto complete failure